From cb1fc9768f2af9da2ca350f4acf37fe29b2bc7d4 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 25 Apr 2023 18:11:04 +0200
Subject: [PATCH 509/954] docs(changelog): add subsections for STM32MP2

Now that STM32MP2 family has different types of SoCs, add relative
subsections for STM32MP2. The same is done for FDTS scopes.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git aa9d394cf63a980d81ce3493fa323c776818e3d1 docs(changelog): add subsections for STM32MP2]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: Iea8820e92d8282fe5e8d39f7f3e12ede6cc5b04a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/356303
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 changelog.yaml | 18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/changelog.yaml b/changelog.yaml
index 89c9be719e..0e9fc66719 100644
--- a/changelog.yaml
+++ b/changelog.yaml
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2021-2023, Arm Limited. All rights reserved.
+# Copyright (c) 2021-2024, Arm Limited. All rights reserved.
 #
 # SPDX-License-Identifier: BSD-3-Clause
 #
@@ -596,6 +596,16 @@ subsections:
           - title: STM32MP2
             scope: stm32mp2
 
+            subsections:
+              - title: STM32MP21
+                scope: stm32mp21
+
+              - title: STM32MP23
+                scope: stm32mp23
+
+              - title: STM32MP25
+                scope: stm32mp25
+
       - title: Texas Instruments
         scope: ti
 
@@ -1261,6 +1271,12 @@ subsections:
             scope: stm32mp2-fdts
 
             subsections:
+              - title: STM32MP21
+                scope: stm32mp21-fdts
+
+              - title: STM32MP23
+                scope: stm32mp23-fdts
+
               - title: STM32MP25
                 scope: stm32mp25-fdts
 
-- 
2.53.0

